common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -65,6 +65,16 @@ trait ABTestSwitches {
     exposeClientSide = true,
   )
 
+  Switch(
+    ABTests,
+    "ab-elements-manager",
+    "Show billboard adverts in merchandising slots to browsers in the variant",
```

Comment:
Grabbed this description from the client-side test, do you think it's suitable?

```suggestion
    "Test serving GEM assets in ad slots on page",
```

Reply:
Yep, thanks for catching this!